aiocache/decorators.py

```diff
@@ -105,7 +106,10 @@ def _key_from_args(self, func, args, kwargs):
 
     async def set_in_cache(self, key, value):
         try:
-            await self.cache.set(key, value, ttl=self.ttl)
+            kwargs = {}
+            if self.ttl is not sentinel:
+                kwargs['ttl'] = self.ttl
```

Comment:
why do we need to replicate the logic here? This is already done in the class so if we just propagate `self.ttl` here (having `sentinel` as default value) it should be enough right?

Reply:
I wanted to avoid having to update all the tests to use `SENTINEL` when mocking the call. Updated.